sense platform/sensors/CSLocationProvider.m

```diff
@@ -192,34 +193,40 @@ - (BOOL) isEnabled {
 	return isEnabled;
 }
 
+// enabled the location provider
 - (void) setIsEnabled:(BOOL) enable {
 	[self enableLocationUpdates:enable];
 	isEnabled = enable;
 }
 
+// notification handler for switching on the locationProvider to make the location provider run
+- (void) setEnabledNotificationHandler: (NSNotification*) notification {
+    NSLog(@"[LocationProvider] setEnabledNotificaiton received!");
+    [self setIsEnabled: TRUE];
```

Comment:
is the convention to use NO and YES always instead of FALSE and TRUE? Or only in specific cases?